testing/src/test/java/at/test/drm/ApplicationIntegrationTest.java

```diff
@@ -1,41 +1,43 @@
 package at.test.drm;
 
-import at.drm.EnableDynamicRelation;
-import at.drm.model.RelationLink;
-import at.drm.service.DynamicRelationsPrintService;
-import at.drm.service.RelationService;
-import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
-import io.zonky.test.db.AutoConfigureEmbeddedDatabase.DatabaseProvider;
```

Comment:
Are you using the right formatter?
If not you should use the projects formatter: https://github.com/Mom0aut/DynamicRelations/blob/master/StyleGuide.xml